ci-operator/step-registry/openshift/selfupgrade/aws/loki/openshift-selfupgrade-aws-loki-workflow.yaml

```
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-selfupgrade-aws-loki
+  steps:
+    pre:
+    - ref: ipi-install-monitoringpvc
+    - ref: ipi-install-loki
+    - chain: ipi-aws-pre-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:latest"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:initial"
+    env:
+      TEST_COMMAND: "run-upgrade"
+      TEST_SUITE: "all"
+      TEST_OPTIONS: "abort-at=100"
```

```suggestion
      TEST_OPTIONS: ""
```
I assumed it means "cancel upgrade", but it really means "start rollback at 100min", that's not needed

---

I haven't looked yet, was just a copy paste from the other upgrade job

---

Right, its being fixed in https://github.com/openshift/release/pull/12544